magenta/music/sequences_lib.py

```diff
@@ -45,6 +44,34 @@ class NegativeTimeException(Exception):
   pass
 
 
+def extract_subsequence(sequence, start_time, end_time):
```

Comment:
I don't know if it's the *right* one, but I think it's the *best* and I'd rather not create a new file at this point.